util/cache/cache.go

```diff
@@ -0,0 +1,57 @@
+package cache
```

Comment:
Usually if we make a new package, we add a `doc.go` file explaining what the package contains (e.g. `Package cache contains an implementation of an in-memory cache with expirable values`).

Reply:
That makes sense, I'll add it in the utility package PR